src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -81,10 +92,13 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
   import dataAccess.driver.api._
 
   // NOTE: if you want to refactor database is inner-class type: this.dataAccess.driver.backend.DatabaseFactory
-  val database = Database.forConfig("", databaseConfig.withUniqueSchema)
+  private val configWithUniqueSchema = databaseConfig.withUniqueSchema
+  val database = Database.forConfig("", configWithUniqueSchema)
 
   // Possibly create the database
   {
+    import SlickDataAccess._
+    log.info(s"Running with database ${configWithUniqueSchema.urlKey} = ${configWithUniqueSchema.urlValue}")
```

Comment:
While setting any property via the url is technically [supported](http://dev.mysql.com/doc/connector-j/en/connector-j-reference-configuration-properties.html#connector-j-reference-url-format), we should *never* be setting `user` and `password` via the `url` in production.

For now, to keep it simple, we should embed the `useSSL` and `requireSSL` in the `url`, and keep using `driver`.

For our Slick/HikariCP/JDBC/MySQL integration, eventually I'd love to refactor and from `driver` to `dataSourceClass` (see doc under [`Database.forConfig`](http://slick.typesafe.com/doc/3.0.0/api/index.html#slick.jdbc.JdbcBackend$DatabaseFactoryDef@forConfig(String,Config,Driver):Database)). At that time we'll also have to from `url` to using `properties.url`, `properties.useSSL` and `properties.requireSSL`. There are also probably a [bunch](https://github.com/brettwooldridge/HikariCP/wiki/MySQL-Configuration) of other values we should tune too.